install/outline-install.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 community-scripts ORG
+# Author: Gerhard Burger (burgerga)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+read -r -p "Enter the public url for your Outline instance (e.g., https://outline.your-domain.tld)): " URL
+read -r -p "Would you like to add Adminer? <y/N> " adminer_prompt
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  sudo \
```

Comment:
mc is missing

Reply:
Is it a convention to include this? I have never used it, and the script doesn't seem to need it.

Reply:
mc is one of our core dependencies. In the next weeks it will be added to build.func